core/src/main/scala/pickling/Pickler.scala

```diff
@@ -26,6 +27,7 @@ trait SPickler[T] {
 
 object SPickler {
   def generate[T]: SPickler[T] = macro Compat.PicklerMacros_impl[T]
+  implicit def spuToSp[T: SPicklerUnpickler]: SPickler[T] = implicitly[SPicklerUnpickler[T]].pickler
```

Comment:
> I agree you should just never have two picklers in scope (ideally) or it's buggy code.

With `SbtSerializer[T]` where this originally came from there was an implicit conversion:

```scala
  implicit def sbtSerializerFromPicklerAndUnpickler[T](implicit pickler: SPickler[T], unpickler: Unpickler[T]): SbtSerializer[T] =
    SbtSerializer[T](pickler, unpickler)
```

so it was easy to have both `SPickler[T]` and `SbtSerializer[T]` in scope. Since `SPickler[T]` won't turn into `SPicklerUnpickler[T]` any more I don't have to worry about it.